yaml/ros2-fastdds-discovery-server.yaml

```diff
@@ -0,0 +1,122 @@
+###############################################
+### ROS 2 Fast-DDS Discovery Primary Server ###
+###############################################
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: discovery-server-primary
+  labels:
+    app: ros2-fastdds-discovery
+    node: server-primary
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      node: server-primary
+  template:
+    metadata:
+      labels:
+        node: server-primary
+    spec:
+      containers:
+      - name: server-primary
+        tty: true
+        image: tomoyafujita/ros:rolling
+        command: ["/bin/bash", "-c"]
+        args: ["source /opt/ros/$ROS_DISTRO/setup.bash && fastdds discovery --server-id 0 --port 11811"]
+        #args: ["source /opt/ros/$ROS_DISTRO/setup.bash && fastdds discovery --server-id 0 --port 11811 --backup"]
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 11811
+          name: primary-port
+      nodeSelector:
+        node-role.kubernetes.io/control-plane: ""
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
```

Comment:
actually this is not required anymore, but dependent on k8s version. just in case, leaving this toleration for the consistency. besides, this toleration does not do anything worse.